platform_api/handlers/jobs_handler.py

```diff
@@ -439,6 +452,7 @@ def convert_job_to_job_response(job: Job) -> dict[str, Any]:
         "id": job.id,
         "owner": job.owner,
         "cluster_name": job.cluster_name,
+        "project_name": job.project_name,
```

Comment:
there won't be such a thing as a non-project job. all jobs will belong to a project. all existing jobs will be assigned to an implicit project named after the base owner.

I don't think we will need to handle this in our old SDK/CLI versions: implicitly all actions will be happening within the user's fallback project.

if you see a use case when this might be problematic, please share.

Reply:
Yes, that's clear.
I haven't found any problematic cases, the comment was mostly re ux of the user, so barely related to this PR, apologies.
thank you for the reply!